lib/mocha.js

```diff
@@ -330,6 +330,9 @@ Mocha.prototype._growl = function(runner, reporter) {
  * mocha.fgrep('foo.');
  */
 Mocha.prototype.fgrep = function(str) {
+  if (!str) {
+    return this;
```

Comment:
or throw `TypeError`

Reply:
maybe, but hard to justify breaking it for that reason.

Reply:
Disagree. I think all _public_ Mocha instance methods should throw when given bad input. Otherwise, no one knows something didn't work because of bad input.